lib/ConvoyLeader/src/App.cpp

```diff
@@ -216,10 +204,10 @@ void App::setup()
                         processingChainFactory.registerLateralSafetyPolicyCreateFunc(LateralSafetyPolicy::create);
 
                         if (false == m_platoonController.init(
-                                         [this](Waypoint& waypoint) { inputWaypointCallback(waypoint); },
-                                         [this](const Waypoint& waypoint) { outputWaypointCallback(waypoint); },
+                                         [this](Waypoint& waypoint) { return inputWaypointCallback(waypoint); },

```

Comment:
Move the lambdas to variables and assign the variables here. Otherwise it looks quite strange.